config/routes.rb

```diff
@@ -2,18 +2,24 @@
 
 Rails.application.routes.draw do
 
+  post '/searchresults/index', to: 'search_results#index'
+  post '/searchresults',       to: 'search_results#index'
+  get '/searchresults/index', to: 'search_results#index'
+  get '/searchresults',       to: 'search_results#index'
```

Comment:
Ich würde wohl den Controller einfach `SearchController` nennen. 

Was macht denn `post` hier? Sollte das nicht immer nur eine Route geben und ausreichen, nämlich `get` auf `index`?

Dann würde hier eine Zeile reichen:

`resources :search`, only: :index

Reply:
Ein `post` ist nur dann notwendig, wenn man Daten in der DB verändern will. Du willst aber nur Daten anfordern.

Reply:
name wurde geändert
posts entfernt
get auf eine Zeile beschränkt
resource line hinzugefügt